### PR TITLE
[MB-5429] CAT III: Remove empty branches

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Remove branches of code that can never be accessed

## Setup
`make server_test`

### To run the linter:
Go to [.golangci.yml](https://github.com/transcom/mymove/blob/51888da54f377fa8acf8865cf576ae5d55d89bd0/.golangci.yml#L46)
Move staticcheck from the disabled list to enabled list
Run `pre-commit run -v --all-files golangci-lint`
Search for any `empty branch` occurrences

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-5429) for this change

### CODE REVIEW

**Review by kimallen — COMMENTED** [review]
How can I run staticcheck to verify if there are any other empty branch errors?

**Comment by Ronolibert** [comment]
@kimallen To enable staticcheck, go to this [file](https://github.com/transcom/mymove/blob/51888da54f377fa8acf8865cf576ae5d55d89bd0/.golangci.yml#L46) and move `staticcheck` from the disabled list to enabled and then run `pre-commit run -v --all-files golangci-lint`. I'll add this to the setup instructions too!

File: cmd/prime-api-dev/scripts/payment_requests.go [hunk 1]
```diff
@@ -666,9 +666,6 @@ func (pr *paymentRequestsData) displayUpdateShipmentMenu() (bool, menuType, erro
 			*/
 
 			err = pr.updateShipmentsJSONToFile(tmpFile, shipmentUpdates, shipmentIndex)
-			if err != nil {
```
Comment (kimallen) [inline cmd/prime-api-dev/scripts/payment_requests.go:669]: I'm wondering if these errors need to actually be handled?  And if we are not going to be handling them, maybe `err` should be `_`.

File: cmd/prime-api-dev/scripts/payment_requests.go [hunk 2]
```diff
@@ -915,9 +912,6 @@ func (pr *paymentRequestsData) displayCreatePaymentRequestMenu() (bool, menuType
 			*/
 
 			err = pr.paymentRequestJSONToFile(tmpFile, serviceItems)
-			if err != nil {
```
Comment (kimallen) [inline cmd/prime-api-dev/scripts/payment_requests.go:918]: Same question here as above

**Review by kimallen — COMMENTED** [review]
I ran  the staticcheck (thanks for adding the instructions), and don't see any remaining empty branches.  Just one question (for two spots) as to whether we should handle the error.  Otherwise, code changes look good!

File: cmd/prime-api-dev/scripts/payment_requests.go (hunk 1 above)
Comment (Ronolibert) [reply cmd/prime-api-dev/scripts/payment_requests.go:669]: Good call!! I added the error handling

File: cmd/prime-api-dev/scripts/payment_requests.go (hunk 2 above)
Comment (Ronolibert) [reply cmd/prime-api-dev/scripts/payment_requests.go:918]: Good call!! I added the error handling

**Review by kimallen — APPROVED** [review]
Changes look good- thx for adding error handling! 
🚢

**Pull request merged**